inlong-sort/sort-connectors/base/src/main/java/org/apache/inlong/sort/base/Constants.java

```diff
@@ -135,6 +135,8 @@ public final class Constants {
 
     public static final String AUTO_DESERIALIZE_FALSE = "autoDeserialize=false";
 
+    public static final String DIRTY_IDENTIFIER_SEPARATOR = "%#%#%#";
```

Comment:
Make sure it is right?

Reply:
it is entirely made up to pass in identifier before the actual dirty data, so it can be anything as long as it wont be user's db/table name